{"tags": ["python", "scipy", "curve-fitting", "sine"], "is_answered": false, "creation_date": 1477322438, "view_count": 30, "score": 0, "last_activity_date": 1477677138, "title": "Scipy curve_fit fails for data with sine function", "answer_count": 1, "owner": {"user_id": 6327699, "reputation": 36, "user_type": "registered", "display_name": "Haminaa", "link": "http://stackoverflow.com/users/6327699/haminaa", "profile_image": "https://www.gravatar.com/avatar/6aad7cad068487fbe21922a03cba9037?s=128&d=identicon&r=PG&f=1"}, "link": "http://stackoverflow.com/questions/40221984/scipy-curve-fit-fails-for-data-with-sine-function", "question_id": 40221984}